eks-anywhere-common/Addons/Partner/Pulumi/pulumi-tester-role.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pulumi-tester-role
```

Comment:
I think it atypical for an object name to have its type as a suffix. In other words, I would name this `pulumi-tester`.  Likewise for the binding.

Reply:
Good call. I do not like smurf naming.